android/KMEA/build.gradle

```diff
@@ -1,8 +1,8 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     repositories {
-        jcenter()
         google()
+        jcenter()
```

Comment:
See https://stackoverflow.com/questions/49839322/errorfailed-to-resolve-android-arch-corecommon1-1-0